src/main/scala/com/exasol/spark/util/ExasolConfiguration.scala

```diff
@@ -51,14 +53,25 @@ final case class ExasolConfiguration(
 
 object ExasolConfiguration {
 
+  val IPv4_DIGITS: String = "(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)"
```

Comment:
Since this rule accepts 192.168.2.**01** as valid address, I don't know if sub-connection could resolve this address (ping does). I'm currently having problem running it tests with my mac so I cannot verify it. Maybe could you please quickly check with our cluster @jpizagno ?